docs/deploy.rst

```diff
@@ -48,6 +48,27 @@ See the documentation on :ref:`network management<network-management>` for more
 
 .. _persistence:
 
+The Deployment Map
+==================
+
+Brownie will maintain a ``map.json`` file in your ``build/deployment/`` folder that lists all deployed contracts on live networks, sorted by chain and contract name.
+
+::
```

Comment:
```suggestion
.. code-block:: json
```

I'm not 100% on if `json` works - but definitely could be `python` or `javascript` - just to ensure proper code highlighting.